### ISSUE TITLE
Is there a reason why Mongo, Redis, Memory, and CouchbaseDB don't implement UserClaimsInterface?

### ISSUE THREAD

**Issue by Gasimzada** [bshaffer/oauth2-server-php#599]

**Comment by bshaffer** [comment]
Nope, I think it's mostly because it's a newer interface, so support just hasn't been added to it yet.

If this is something you'd like to see, you can submit a PR, or we can leave this issue open to gauge how interested the community is in having this added.

**Comment by Gasimzada** [comment]
For now, I just copied the user claims functions from Pdo/Cassandra/DynamoDB and added UserClaimsInterface to Mongo. Everything seems to be working. I don't quiet understand the point of user claims yet (I am a newbie at OIDC) but thinking that others use the exact same functions for user claims, copying seemed like a logical decision.

I will read the specs more to understand it and try to do a pull request.

**Comment by bshaffer** [comment]
That would be lovely! User claims is used in the UserInfoCo troller
